code/modules/mob/living/living_health_procs.dm

```diff
@@ -75,14 +75,11 @@
 /mob/living/proc/updateStamina(feedback = TRUE)
 	if(staminaloss < maxHealth * 1.5)
 		return
-	switch(knocked_down)
-		if(0)
-			if(feedback)
-				visible_message("<span class='warning'>\The [src] slumps to the ground, too weak to continue fighting.</span>",
-					"<span class='warning'>You slump to the ground, you're too exhausted to keep going...</span>")
-			knock_down(4)
-		if(1 to 3)
-			set_knocked_down(4, FALSE)
+	if(!IsKnockdown())
+		if(feedback)
+			visible_message("<span class='warning'>\The [src] slumps to the ground, too weak to continue fighting.</span>",
+				"<span class='warning'>You slump to the ground, you're too exhausted to keep going...</span>")
+	Knockdown(80)
```

Comment:
Perhaps use `AmountKnockdown` here? The idea is to keep the mob down until the stamina regens, but not long past that.

Reply:
Knockdown(x) is knockdown = min(x, currentknockdown)

Reply:
This works exactly as it did before/

Reply:
Ah, didn't notice `Knockdown(80)` is not inside the `if(!IsKnockdown())` check.
Alright.